docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.general.scss

```diff
@@ -1,5 +1,5 @@
 $hb-global-theme-list: ('airy', 'colorful', 'traditional');
-$hb-colorful-theme-variation-list: ('colorful-teal', 'colorful-mint');
```

Comment:
It looks like this isn't being used anywhere, right? (I searched the codebase and found nothing)

Let's delete it then.